conf/modules.config

```diff
@@ -244,14 +244,12 @@ process {
     withName: "BWA.*MEM" {
         ext.args         = { meta.status == 1 ? '-K 100000000 -M -B 3' : '-K 100000000 -M' }
         ext.prefix       = { params.split_fastq > 1 ?  "${meta.id}".concat('.').concat(reads.get(0).name.findAll(/part_([0-9]+)?/).last()) : "" }
+        publishDir       = [
+            enabled: false
```

Comment:
Isn't this the default? I think we discuss this in one of the previous PR'S?

Reply:
Wasn't working properly actually

Reply:
oh ok, so should be added back then to the other modules? Or is only BWA not working?

Reply:
no idea, I'll need to do some more proper tests with that

Reply:
Is this solved?

Reply:
still not, I have no idea why I have the bam if I don't have this enabled: false here